plugins/flytekit-async_fsspec/README.md

```diff
@@ -0,0 +1,14 @@
+# Flytekit Async fsspec Plugin
+
+The Flyte async fsspec plugin is a powerful addition to the Flyte ecosystem designed to optimize the performance of object transmission. This plugin focuses on overriding key methods of the file systems in fsspec to introduce efficiency improvements, resulting in accelerated data transfers between Flyte workflows and object storage.
+
+Currently, the async fsspec plugin improves the following file systems:
+1. s3fs
+
+To install the plugin, run the following command:
+
+```bash
+pip install flytekitplugins-async_fsspec
+```
+
+Once installed, the plugin will automatically override the original file system and register optimized ones, seamlessly integrating with your Flyte workflows.
```

Comment:
FYI @pryce-turner

Reply:
Thanks! So far I was just building an image with Troy's branch in his fork, but I'll install the plugin as well.